cg/src/domains/management/management.routes.js

```diff
@@ -99,6 +99,8 @@ module.exports = app => {
     asyncHandler(async (req, res) => subjectsController.getRectificationRequest(req, res))
   );
 
+  router.get('/events', asyncHandler(async (req, res) => statsController.events(req, res)));
```

Comment:
Change to /past-events, that way it's clear we are talking about the past events and not present events

Reply:
I do not think that it makes to concentrate attention that events are "past". This is http endpoint so it by definition can return only "past" events, so this word in the endpoint name is redundant.

Reply:
An uniformed reader would not know that. I vote for clarity

Reply:
Would not know what? The fact that HTTP endpoint provides access to collection of some entities? Why do not we name all endpoints in the same way - `/api/management/created-processors`, `/api/management/existed-subjects` or something like this?

Btw, about endpoint names. Are we trying to build restful api or not? At this moment it looks more like not is an answer. An example

```
GET /api/management/processors/list
POST /api/management/processors/add
POST /api/management/processors/update
POST /api/management/processors/delete
```

The proper implementation is to use the same endpoint `/api/management/processors` with GET, POST, PUT and DELETE methods. Parts `list`, `add`, `update` and `delete` do not provide any additional clarity.

Reply:
Ok you have a point about the events. 
About the API, it wasn't me who made the decision to make it like this. But in my opinion it's better to have the actual action in the endpoint, makes it easier to read. Not everything fits in GET, POST, PUT and DELETE. Anyways, would be unnecessary work to change it now

Reply:
I do not call to change it right now, I just say what is the proper way. I agree that not everything fits in CRUD operations, but, example is 100% CRUD and operation names in endpoints are redundant. Just to consider it as something that can be improved later.